### PR TITLE
Add kinjuplus's story

### PR DESCRIPTION
**Pull request by kinjuplus**
2015/02/27 kinjuplus add

### CODE REVIEW

**Comment by kinjuplus** [comment]
kinjuplus try to create his own adventure story for practice purpose

**Comment by kinjuplus** [comment]
Please review this, @casey-collab

File: Chinese/YingJeouMa.md [hunk 1]
```diff
@@ -0,0 +1 @@
+成為某國總統，禍國殃民。
```
Comment (casey-collab) [inline Chinese/YingJeouMa.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: Chinese/大话西游.md [hunk 2]
```diff
@@ -22,4 +22,6 @@
 
 [唐僧](hatetong/hatetong.md)？
 
-[恭喜您](congrats/congrats.md)
\ No newline at end of file
+[恭喜您](congrats/congrats.md)
+
+[馬英九](YingJeouMa/YingJeouMa.md)?
```
Comment (casey-collab) [inline Chinese/大话西游.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Chinese/YingJeouMa.md (hunk 1 above)
Comment (casey-collab) [inline Chinese/YingJeouMa.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**